pandas/tests/arrays/period/test_arrow_compat.py

```diff
@@ -69,6 +71,9 @@ def test_arrow_array_missing():
     assert result.storage.equals(expected)
 
 
+@pytest.mark.xfail(
+    pa_version_under2p0, reason="pyarrow incorrectly uses pandas internals API"
```

Comment:
do we have an issue for this? or is it just a bug in pyarrow that is subsequently fixed?

Reply:
more recent versions use the pseudo-public API